dao/src/main/scala/za/co/absa/enceladus/dao/rest/RestDaoFactory.scala

```diff
@@ -21,8 +21,8 @@ object RestDaoFactory {
 
   private val restTemplate = RestTemplateSingleton.instance
 
-  def getInstance(authCredentials: MenasCredentials, apiBaseUrls: List[String]): MenasRestDAO = {
-    val apiCaller = CrossHostApiCaller(apiBaseUrls)
+  def getInstance(authCredentials: MenasCredentials, apiBaseUrls: List[String], urlsTryCounts: List[Int]): MenasRestDAO = {
```

Comment:
I was considering making this 
```
  def getInstance(authCredentials: MenasCredentials, apiBaseUrls: List[String], urlsTryCounts: List[Int] = List.empty()): MenasRestDAO = {
```
but decided against it. To make the possibility more clear. But still not sure, I can be easily persuaded to do so 😉 .